src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -0,0 +1,250 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols.Retargeting;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
+{
+    public class ReadOnlyStructsTests : CompilingTestBase
+    {
+        [Fact()]
+        public void WriteableInstanceAutoPropsInRoStructs()
+        {
+            var text = @"
+public readonly struct A
+{
+    // ok
+    int ro => 5;
```

Comment:
I need to add another case though where we definitely add a backing field.

---
In reply to: [121477343](https://github.com/dotnet/roslyn/pull/20142#discussion_r121477343) [](ancestors = 121477343,121437834)